### ISSUE TITLE
Trouble generating brigge file for certain objects

### ISSUE THREAD

**Issue by axeway22** [significant-bit/brigge#1]
Hello. I just bought the add on today for blender. Everything works great when importing static meshes like buildings and roads in a scene. However, when I am trying to export a car model, only the .brigge file is generated, and no meshes.

Am I doing something wrong here? for the car in blender, I first seperated all objects of the car by materials, and then selected all the objects of the car and tried to export, but only the brigge file was generated, thereby unable to be opened in Unreal.

**Comment by significant-bit** [comment]
It should export all selected mesh objects 🤔 so let's figure out what is going wrong.

Does the .brigge file from the car have any text describing the car pieces? As in _"mesh named 'wheels' at (x,y,z) ..."_

Brigge does support multiple materials per object, so separating the parts might be extra work. Unless that helps with some other part of your project.

**Comment by axeway22** [comment]
Nope. The file is simply the name I gave it while saving the brigge file. 
I also tried exporting individual objects of the car, like the hood only for example but it doesnt work. Strange thing is that for all other objects in my scene, it exported flawlessly and opened flawlessly in Unreal.

Could  the issue be due to some objects using complicated procedural materials? or perhaps it has problems with objects made from non-primitive base meshes? I will try to run some more tests today. In case you want me to check some parameters in Blender before exporting, please let me know.

**Update - Issue Solved:** 
Looks like naming of  the meshes in blender was preventing the export. All my meshes had names with "LOD" in them.
![image](https://user-images.githubusercontent.com/48159851/102839855-75cc9380-443c-11eb-99e1-8fd21498c428.png)

So after I renamed all the meshes with something more understandable, like body, tires, rims, etc. then it worked perfectly!

**Comment by significant-bit** [comment]
Ah interesting! When Brigge sees LOD1 in the name it looks for the other levels of detail like LOD0, LOD2, and so on. Since it can't find LOD0 (or just the base name like "chassis") it skipped the whole thing.

To fix: it should either export orphaned LODs as regular meshes, or alert you that something is missing.
